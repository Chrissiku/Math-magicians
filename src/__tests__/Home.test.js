import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('Home component testing with snapshot', () => {
  const TREE = renderer.create(<Home />).toJSON();

  expect(TREE).toMatchSnapshot();
});
