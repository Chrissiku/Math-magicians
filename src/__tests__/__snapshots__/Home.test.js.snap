// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing with snapshot 1`] = `
<section>
  <h1
    className="title"
  >
    Welcome to our page !
  </h1>
  <p
    className="text"
  >
    A calculator is a device that performs arithmetic operations on numbers. Basic calculators can do only addition, subtraction, multiplication and division mathematical calculations. However, more sophisticated calculators can handle exponential operations, square roots, logarithms, trigonometric functions and hyperbolic functions. Internally, some calculators perform all these functions by repeated addition processes.
  </p>
  <p
    className="text"
  >
    Online calculator for quick calculations, along with a large collection of calculators on math, finance, fitness, and more, each with in-depth information.
  </p>
</section>
`;
